### ISSUE TITLE
aspnetcore-runtime-2.2 not getting installed

### ISSUE THREAD

**Issue by marceletchart** [dotnet/core#2752]
Problem encountered on https://dotnet.microsoft.com/download/linux-package-manager/ubuntu18-04/sdk-current
Operating System: Linux Ubuntu 18.04 - x64

Provide details about the problem you are experiencing. Include your operating system version, exact error message, code sample, and anything else that is relevant.
I got an error after following the instructions on the above website. 
The error message is as follows: 
The following packages have unmet dependencies:
 dotnet-sdk-2.2 : Depends: aspnetcore-runtime-2.2 (>= 2.2.4) but it is not going to be installed
                  Depends: dotnet-runtime-2.2 (>= 2.2.4) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.

**Comment by karelz** [comment]
@dagood can you please help here? This looks like a familiar failure ...

**Comment by leecow** [comment]
This is caused by a system dependency not being found by the dependency package installer. Could you `sudo apt-get install dotnet-runtime-deps-2.2`? This will tell us exactly which package it's not finding in the Ubuntu repos.

**Comment by marceletchart** [comment]
@leecow 
Running the install for dotnet-runtime-deps-2.2 yields the following:
The following packages have unmet dependencies:
 dotnet-runtime-deps-2.2 : Depends: libssl1.0.0 but it is not installable
                           Depends: libicu60 but it is not installable
E: Unable to correct problems, you have held broken packages.

**Comment by dagood** [comment]
`libicu60` and `libssl1.0.0` should be available in Ubuntu 18.04... can you run these diagnosis steps and share the results?

```
# Shows the repositories you have configured. Source: https://askubuntu.com/a/741948
grep -r --include '*.list' '^deb ' /etc/apt/sources.list /etc/apt/sources.list.d/
# Show your distro version.
cat /etc/os-release
```

**Comment by karelz** [comment]
Not actionable, closing. Feel free to reopen if there is more info available.
